src/HTSeq/_HTSeq.pyx

```diff
@@ -662,15 +662,19 @@ cdef class Sequence( object ):
          new_name = self.name + "[part]"
       return Sequence( self.seq[ item ], new_name )
    
-   def write_to_fasta_file( self, fasta_file ):
+   def write_to_fasta_file( self, fasta_file, bases_per_line=70 ):
```

Comment:
@iosonofabio I didn't use characters as I found it could be misleading and lead someone to think it also applies to the header. Updated as requested.